### ISSUE TITLE
Video cannot be played

### ISSUE THREAD

**Issue by G0WiththeFloW** [kaungsatthe1n/Tako-Play#13]
Hello @DDavidPrime and @kaungsatthe1n recently started using your apk. It's awesome loved it.
But recently it's giving an error like video can't be played.

![IMG_20220309_152712](https://user-images.githubusercontent.com/101253338/157418137-590e7069-e5d2-409d-9243-678f456417ec.jpg)
Hope you will look into this error asap.

Once again thanks for the app

**Comment by dacher996** [comment]
Hi @G0WiththeFloW . I am really glad that you like the app and I would agree with you - it definitely is awesome!

I'd like to mention that I am in no way affiliated with this project. I just like checking it out periodically and eventually providing some help. However, ALL of the credits definitely go to @kaungsatthe1n for making it possible for people around the world to watch their favourite anime as easy as this.

That being said, whenever I have time I will gladly look into this and many other issues, striving to help build a better experience. But otherwise when I am unable to do this, I think the best thing would be waiting a response from @kaungsatthe1n since they own this project.

---
Oh yeah, @DDavidPrime is my other account, so don't be confused by that. 😄

**Comment by kaungsatthe1n** [comment]
@G0WiththeFloW Yes, I got a lot of request about this issue from my e-mail  and  feel sorry for about this.
i hope i can fix it within this month or maybe next month.
i suggest you guys to use webview player if normal video player is not working properly.
